tests/test_definitions/column_map_expectations/expect_column_values_to_be_in_set.json

```diff
@@ -239,7 +239,7 @@
     },
     {
       "title": "basic_negative_test_case_datetime_set",
-      "suppress_test_for": ["sqlite", "bigquery"],
+      "suppress_test_for": ["sqlite", "bigquery", "trino"],
```

Comment:
Don't suppress this datetime test for Trino, as we will be working with their engineers to get this to work